desktop/mac/release-notes/index.md

```diff
@@ -29,9 +29,41 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Security
+
+- Fixed CVE-2021-45449 that affects users currently on Docker Desktop version 4.3.0 or 4.3.1
+
+### New
+
+- Easy, Secure sign in with Auth0 and Single Sign-on
```

Comment:
This is unclear IMHO.
`Single Sign-On on docker Hub`
will be more direct.

Reply:
Perhaps we should link to the documentation for that; https://docs.docker.com/single-sign-on/ (I think)

Reply:
>  Single Sign-On on hub.docker.com

It’s not just SSO, the username/password flow has changed as well from the in-app dialog to a web page.

Reply:
Added the link to the docs!